lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -839,6 +839,7 @@ def _create_deployment_parameters(
             azure_node_runbook = node_space.get_extended_runbook(
                 AzureNodeSchema, type_name=AZURE
             )
+            azure_node_runbook.subscription_id = self.subscription_id
```

Comment:
Add comments above to explain the sig need it.

Reply:
Added comment in the `AzureNodeSchema` class

Reply:
I mean here needs a comment that, the subscription id may be used by sig. So it needs to be in node level.

Reply:
Got it! Thanks for clarifying.